.github/workflows/workflow-java.yaml

```diff
@@ -62,15 +50,20 @@ jobs:
           echo "Is github.event_name != 'pull_request': ${{ github.event_name != 'pull_request' }}"
           echo "=== Check environment conditionals"
           echo "inputs.do_staging_build: ${{ inputs.do_staging_build }}"
-          echo "inputs.do_production_build: ${{ inputs.do_production_build }}"
   # Test projects
   test_mvn:
+    # If we are in a commit from the [actions] pipeline (see java-java-update-version.yaml), do nothing
+    if: |
+      !contains(github.event.head_commit.message, '[actions]')
```

Comment:
I would wrap this in `${{ }}`, even though it's not strictly needed. It's more clear, and IntelliJ will highlight it correctly. (Also maybe `>-`, since the newlines aren't needed.)

Reply:
> I would wrap this in ${{ }}, even though it's not strictly needed. It's more clear, and IntelliJ will highlight it correctly 

Can you provide a suggestion? I'm unsure how to wrap it and why

> Also maybe >-, since the newlines aren't needed. 

Do you mean that `|` is not needed? I've tried that (`if: !contains( ....`) but github actions didn't like it